### ISSUE TITLE
Unknown keyword: KRNUMX

### ISSUE THREAD

**Issue by hakonhagland** [OPM/opm-common#3053]
I am trying to implement directional relative permeabilities in flow. To start with I just modified [SPE1CASE2.DATA](https://github.com/OPM/opm-tests/blob/master/spe1/SPE1CASE2.DATA) like this

```
-- ....

RUNSPEC

-- ....

SATOPTS
 'DIRECT'
/
 
-- ....

REGIONS

KRNUMX
 300*1/

KRNUMY
 300*1/
 
KRNUMZ
 300*1/
-- ....
```

But this gives the following error:
```
$ ../opm-simulators/build/bin/flow SPE1CASE2_DIAGREL.DATA
hwloc/linux: Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
hwloc/linux: Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
**********************************************************************
*                                                                    *
*                      This is flow 2022.10-pre                      *
*                                                                    *
* Flow is a simulator for fully implicit three-phase black-oil flow, *
*             including solvent and polymer capabilities.            *
*          For more information, see https://opm-project.org         *
*                                                                    *
**********************************************************************

Using 1 MPI processes with 2 OMP threads on each 

Reading deck file 'SPE1CASE2_DIAGREL.DATA'
    0 Reading RUNSPEC  in SPE1CASE2_DIAGREL.DATA line 17
    1 Reading TITLE    in SPE1CASE2_DIAGREL.DATA line 20
    2 Reading DIMENS   in SPE1CASE2_DIAGREL.DATA line 23
    3 Reading EQLDIMS  in SPE1CASE2_DIAGREL.DATA line 28
    4 Reading TABDIMS  in SPE1CASE2_DIAGREL.DATA line 33
    5 Reading SATOPTS  in SPE1CASE2_DIAGREL.DATA line 36
    6 Reading OIL      in SPE1CASE2_DIAGREL.DATA line 40
    7 Reading GAS      in SPE1CASE2_DIAGREL.DATA line 41
    8 Reading WATER    in SPE1CASE2_DIAGREL.DATA line 42
    9 Reading DISGAS   in SPE1CASE2_DIAGREL.DATA line 43
   10 Reading FIELD    in SPE1CASE2_DIAGREL.DATA line 48
   11 Reading START    in SPE1CASE2_DIAGREL.DATA line 50
   12 Reading WELLDIMS in SPE1CASE2_DIAGREL.DATA line 53
   13 Reading UNIFIN   in SPE1CASE2_DIAGREL.DATA line 64
   14 Reading UNIFOUT  in SPE1CASE2_DIAGREL.DATA line 65
   15 Reading GRID     in SPE1CASE2_DIAGREL.DATA line 67
   16 Reading INIT     in SPE1CASE2_DIAGREL.DATA line 74
   17 Reading NOECHO   in SPE1CASE2_DIAGREL.DATA line 77
   18 Reading DX       in SPE1CASE2_DIAGREL.DATA line 79
   19 Reading DY       in SPE1CASE2_DIAGREL.DATA line 82
   20 Reading DZ       in SPE1CASE2_DIAGREL.DATA line 85
   21 Reading TOPS     in SPE1CASE2_DIAGREL.DATA line 89
   22 Reading PORO     in SPE1CASE2_DIAGREL.DATA line 93
   23 Reading PERMX    in SPE1CASE2_DIAGREL.DATA line 97
   24 Reading PERMY    in SPE1CASE2_DIAGREL.DATA line 101
   25 Reading PERMZ    in SPE1CASE2_DIAGREL.DATA line 105
   26 Reading ECHO     in SPE1CASE2_DIAGREL.DATA line 109
   27 Reading PROPS    in SPE1CASE2_DIAGREL.DATA line 111
   28 Reading PVTW     in SPE1CASE2_DIAGREL.DATA line 114
   29 Reading ROCK     in SPE1CASE2_DIAGREL.DATA line 127
   30 Reading SWOF     in SPE1CASE2_DIAGREL.DATA line 134
   31 Reading SGOF     in SPE1CASE2_DIAGREL.DATA line 153
   32 Reading DENSITY  in SPE1CASE2_DIAGREL.DATA line 168
   33 Reading PVDG     in SPE1CASE2_DIAGREL.DATA line 178
   34 Reading PVTO     in SPE1CASE2_DIAGREL.DATA line 199
   35 Reading REGIONS  in SPE1CASE2_DIAGREL.DATA line 229

Error: Unknown keyword: KRNUMX

Error: Unrecoverable errors while loading input: Problem with keyword 
In <memory string> line 0
Unknown keyword: KRNUMX

```

**Comment by bska** [comment]
@hakonhagland : As your PR #3054 addresses the underlying issue, I assume that this ticket can be closed?

**Comment by hakonhagland** [comment]
Yes this has be resolved by #3054.
